### PR TITLE
Fix obviously wrong (but non significant) error in dictionaries

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Introduced in #23916

### CODE REVIEW

File: tests/queries/0_stateless/02832_alter_max_sessions_for_user.sh [hunk 1]
```diff
@@ -23,14 +23,14 @@ function test_alter_profile()
 
     ${CLICKHOUSE_CLIENT} -q $"ALTER SETTINGS PROFILE ${PROFILE} SETTINGS max_sessions_for_user = ${max_session_count}"
 
-    # Create sesssions with $max_session_count resriction
+    # Create sessions with $max_session_count restriction
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/02832_alter_max_sessions_for_user.sh:26]: No surprise that the test is flaky.

File: src/Dictionaries/IDictionary.h [hunk 2]
```diff
@@ -109,6 +109,9 @@ class IDictionary : public IExternalLoadable, public IKeyValueEntity
 
     virtual size_t getQueryCount() const = 0;
 
+    /// The percentage of time a lookup successfully found an entry.
+    /// When there were no lookups, it returns zero (instead of NaN).
```
Comment (alexey-milovidov) [inline src/Dictionaries/IDictionary.h:113]: Life-saving comment №1.

File: src/Dictionaries/IDictionary.h [hunk 3]
```diff
@@ -109,6 +109,9 @@ class IDictionary : public IExternalLoadable, public IKeyValueEntity
...
+    /// The value is calculated non atomically and can be slightly off in the presence of concurrent lookups.
```
Comment (alexey-milovidov) [inline src/Dictionaries/IDictionary.h:114]: Life-saving comment №2.

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit 2688a6268caef7b7430a6a7ffc6aa057c1b4360a with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60005/2688a6268caef7b7430a6a7ffc6aa057c1b4360a/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/60005/2688a6268caef7b7430a6a7ffc6aa057c1b4360a/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Pull request merged**